frontend/components/home.jsx

```diff
@@ -61,6 +62,11 @@ class Home extends React.Component {
     this.setState({ soundPlaying: !this.state.soundPlaying });
   }
 
+  handleSongFinishedPlaying() {
+    this.setState({currentTrackURL: this.state.currentTrackURL});
```

Comment:
add spaces at the end and beginning of the object, reference the style in the setState above